test_regression/test_doc_samples/test_using_plotly.py

```diff
@@ -0,0 +1,40 @@
+from module_helper import *
+from pathlib import Path
+from constants import test_files_path
+
+
+def test_using_plotly(tmp_path):
+    input_folder = Path(test_files_path) / "doc_samples" / "plotly"
+
+    # === Sample code
+    from meshlib import mrmeshpy as mm
+    from meshlib import mrmeshnumpy as mn
+    import numpy as np
+    import plotly.graph_objects as go
+
+    # load mesh
+    mesh = mm.loadMesh(mm.Path(str(Path(input_folder / "fox_geometrik.stl"))))
+    # extract numpy arrays
+    verts = mn.getNumpyVerts(mesh)
+    faces = mn.getNumpyFaces(mesh.topology)
+    # print(verts)
+
+    # prepare data for plotly
+    vertsT = np.transpose(verts)
+    facesT = np.transpose(faces)
+
+    # draw
+    fig = go.Figure(data=[
```

Comment:
doesnt it freeze on this line?

Reply:
This string doesn't render any object. It just creates object.
Showing object in browser happens in fig.show() that removed from this sample to avoid such issue.
It makes little sense to do it in test.